src/ray/rpc/grpc_server.h

```diff
@@ -36,6 +36,15 @@ namespace rpc {
           #SERVICE ".grpc_server." #HANDLER));                                  \
   server_call_factories->emplace_back(std::move(HANDLER##_call_factory));
 
+#define RPC_SERVICE_HANDLER_WITH_BACK_PRESSURE(SERVICE, HANDLER, BACK_PRESSURE_NUM) \
```

Comment:
Consider consolidating this with the original, and explicitly passing a limit of -1 for services that don't support backpressure.